### PR TITLE
added range fetching tracer option

### PR DESCRIPTION
**Pull request by jeongkyun-oh**
## Proposed changes

- In case of range fetching, this PR add an option to choose tracer either `fastCallTracer` (default) or `callTracer`. This option is for the unexpected circumstance when `fastCallTracer` is failed to reproduce data using javascript `callTracer`.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- https://github.com/klaytn/klaytn/issues/1531

### CODE REVIEW

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -998,6 +998,12 @@ var (
 		Value:  chaindatafetcher.DefaultMaxProcessingDataSize,
 		EnvVar: "KLAYTN_CHAINDATAFETCHER_MAX_PROCESSING_DATA_SIZE",
 	}
+	ChainDataFetcherRangeFetchingTracerFlag = cli.StringFlag{
+		Name:   "chaindatafetcher.rangefetching.tracer",
+		Usage:  "Chaindatafetcher range fetching tracer (callTracer or fastCallTracer)",
+		Value:  chaindatafetcher.FastCallTracer,
```
Comment (kjhman21) [inline cmd/utils/flags.go:1004]: I'm just curious. Wouldn't it be better to use integer value instead of string?

Comment (jeongkyun-oh) [reply cmd/utils/flags.go:1004]: With debug trace API, the tracer should be passed with string. This seems simpler compared to integer and converting it to string or vice versa.

File: cmd/utils/flags.go [hunk 2]
```diff
@@ -998,6 +998,12 @@ var (
 		Value:  chaindatafetcher.DefaultMaxProcessingDataSize,
 		EnvVar: "KLAYTN_CHAINDATAFETCHER_MAX_PROCESSING_DATA_SIZE",
 	}
+	ChainDataFetcherRangeFetchingTracerFlag = cli.StringFlag{
+		Name:   "chaindatafetcher.rangefetching.tracer",
+		Usage:  "Chaindatafetcher range fetching tracer (callTracer or fastCallTracer)",
```
Comment (yoomee1313) [inline cmd/utils/flags.go:1003]: What about saying clearly that only two tracers are available when range fetching chaindatafetcher?
```suggestion
		Usage:  "Chaindatafetcher range fetching tracer (only callTracer or fastCallTracer is available)",
```

**Review by yoomee1313 — COMMENTED** [review]
@jeongkyun-oh it looks good to me except the other comments and conflicts. Could you resolve the conflicts?

**Comment by jeongkyun-oh** [comment]
Closed due to the priority.

**Pull request closed without merging**